docs/RELEASE_NOTES.md

```diff
@@ -45,6 +45,9 @@
     now treat the values of the optional fields `isSearchable`, `inputHint` and `attributeConstraint` 
     as (`true`, `SingleLine` and `None` respectivley) if they are `null` or not passed. [#354](https://github.com/commercetools/commercetools-sync-java/issues/354)
 
+- 🛠️ **Enhancements** (1)
+    - **Commons** - Benchmarks are now run once on every merge to `master` with a lower number of resources for faster benchmarking. [#246](https://github.com/commercetools/commercetools-sync-java/issues/246)
```

Comment:
I get that it will be useful for a maintainer, but not for a lib user.

hmm, the problem with that Is the graph would be piled up quickly with new results for every single PR opened. 

Also shouldn't the graph show the benchmarks of the actual state of the library (on master) for the users using it as opposed to just a PR that a dev is working on?

> How long does it take now?

Still didn't try a full benchmark run on travis with this change. We'll see after this is merged.

Reply:
> I get that it will be useful for a maintainer, but not for a lib user

Any contributor would probably use the library too - stable performance is relevant to everyone.

> hmm, the problem with that Is the graph would be piled up quickly with new results for every single PR opened

> Also shouldn't the graph show the benchmarks of the actual state of the library (on master) for the users using it as opposed to just a PR that a dev is working on?

Maybe run the benchmark on any PR, report readable regression but update benchmark JSON only on master. wdyt?

Reply:
This would be good, yes. Will add to the improvements issue #246 and address later then ;)

Reply:
https://github.com/commercetools/commercetools-sync-java/issues/246#issuecomment-448558433